### PR TITLE
Add clear button back into auto organize page

### PR DESCRIPTION
**Pull request by Nodja**
The clear button in the auto-organize page was removed during the dashboard redesign, this change restores it.

### CODE REVIEW

File: MediaBrowser.WebDashboard/dashboard-ui/autoorganizelog.html [hunk 1]
```diff
@@ -18,8 +18,8 @@
                     <div class="listTopPaging" style="float: left; position: relative; top: 15px;">
                     </div>
 
-                    <!--<button style="display: none;" class="btnClearLog" type="button" data-inline="true" data-icon="forbidden" data-mini="true">Clear</button>-->
                     <div style="float: right; position: relative; top: 15px;margin-top: -5px;display:none;" class="organizeTaskPanel">
+                        <button type="button" class="btnClearLog" data-icon="forbidden" data-mini="true" data-inline="true" style="display: none;">Clear</button>
```
Comment (TheTedAdams) [inline MediaBrowser.WebDashboard/dashboard-ui/autoorganizelog.html]: This isn't localizable. Should probably be a new string added to the files in https://github.com/Nodja/MediaBrowser/tree/patch-2/MediaBrowser.Server.Implementations/Localization/Server

**Comment by Nodja** [comment]
Added localization and made the button localizable.

File: MediaBrowser.Server.Implementations/Localization/Server/sv.json [hunk 2]
```diff
@@ -1,4 +1,4 @@
-{
+
```
Comment (TheTedAdams) [inline MediaBrowser.Server.Implementations/Localization/Server/sv.json]: You accidentally deleted the opening bracket of this json object

**Comment by LukePulverenti** [comment]
merged manually.

**Pull request closed without merging**